### DISCUSSION TITLE
Pause 3D Printers

### DISCUSSION THREAD

**Discussion by petelton** [community/community#133765] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello.
I am facing on particular issue with my 3D Printer when I try to use the code "Pause at height", not works..
Anyone are facing the same problem?
How can I fix it?
Thanks

**Comment by queenofcorgis** [comment]
Hi @petelton ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your brand of 3D printer. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
